lib/rentlinx/errors.rb

```diff
@@ -7,4 +7,34 @@ class UnexpectedAttributes < RentlinxError
 
   class InvalidTypeParam < RentlinxError
   end
+
+  class BadRequest < RentlinxError
+    def initialize
+      super('The request sent to the server was invalid.')
+    end
+  end
+
+  class NotFound < RentlinxError
```

Comment:
Also, for these exceptions, it's often useful to include the response object as part of the error so that they can be investigated in rescue blocks.